### ISSUE TITLE
WCAG 2 Layers of Guidance still refers to WCAG 2.0

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag21#957]
At [https://www.w3.org/TR/WCAG21/#wcag-2-layers-of-guidance](url) I see two references to WCAG 2.0.

Shouldn't this be adjusted to 2.1?

> Success Criteria - For each guideline, testable success criteria are provided to allow **WCAG 2.0** to be used

> Sufficient and Advisory Techniques - For each of the guidelines and success criteria in the **WCAG 2.0** document itself,

And same for the following link which goes to a 2.1 URL: 

> Where common failures are known, these are also documented. [ See also Sufficient and Advisory Techniques in Understanding WCAG 2.0.](url).

**Comment by jake-abma** [comment]
another one can be seen at: https://www.w3.org/TR/WCAG21/#wcag-2-1-supporting-documents

> See Web Content Accessibility Guidelines (WCAG) Overview for a description of the WCAG 2.0 supporting material

**Comment by michael-n-cooper** [comment]
This issue was moved to w3c/wcag#411
